cupy/cuda/function.pyx

```diff
@@ -146,11 +146,11 @@ cdef _launch(intptr_t func, Py_ssize_t grid0, int grid1, int grid2,
     if enable_cooperative_groups:
         driver.launchCooperativeKernel(
             func, <int>grid0, grid1, grid2, <int>block0, block1, block2,
-            <int>shared_mem, stream, <intptr_t>&(kargs[0]))
+            <int>shared_mem, stream, <intptr_t>kargs.data())
     else:
         driver.launchKernel(
             func, <int>grid0, grid1, grid2, <int>block0, block1, block2,
-            <int>shared_mem, stream, <intptr_t>&(kargs[0]), <intptr_t>0)
```

Comment:
Function args will never be empty, double-checked.

Reply:
Just curious, even when we use a RawKernel like this?
```cuda
__global__ void f() { /* ... */ }
```

Reply:
I was discussing with @asi1024 such use case the other day.
Is there any case that someone would want to do this?
We couldn't find any.

Reply:
I can only think of very pathological cases of no practical uses 😂 I think it's safe to merge this!